modules/nodes-cma-autoscaling-custom-trigger-cron.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * nodes/cma/nodes-cma-autoscaling-custom-trigger.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-cma-autoscaling-custom-trigger-cron_{context}"]
+= Understanding the cron trigger
+
+You can scale pods based on a time range. 
+
+When the time range starts, the custom metrics autoscaler scales the pods associated with an object from the configured minimum number of pods to the specified number of desired pods. At the end of the time range, the pods are scaled back to the configured minimum. The time period must be configured in link:https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#writing-a-cronjob-spec[cron format]. 
+
+[NOTE]
+====
+The custom metrics autoscaler with cron trigger scales pods based only on the specified times and cannot scale pods on a daily, weekly, monthly, or yearly schedule.
+====
+
+The following example scales the pods associated with this scaled object from `0` to `100` from 6:00 a.m to 6:30 p.m India Standard Time.
```

Comment:
Per IBM word usage ([AM](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a) and [PM](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p)), the times should be in caps and without periods:
```suggestion
The following example scales the pods associated with this scaled object from `0` to `100` from 6:00 AM to 6:30 PM India Standard Time.
```

Reply:
@skopacz1 Thank you! They must have changed this one on me. I have the IBM style guide in book form, which I did consult here. I guess I need a new one!